pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -1075,6 +1099,18 @@ public void setTransactionIsolation(int level) throws SQLException {
     }
   }
 
+  protected String getSessionIsolationQuery(int level) {
+    String levelName = getIsolationLevelName(level);
+    if (levelName == null) {
+      levelName = DEFAULT_ISOLATION_LEVEL_NAME;
+    }
```

Comment:
Does this change the current behaviour? Currently we throw an exception if the leveName is null. https://github.com/pgjdbc/pgjdbc/blob/3c3e5b62349c42d9d3dfe1fe0102776d4de8c353/pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java#L1061

Reply:
This method wan't used at all, removed it